src/dosbox.cpp

```diff
@@ -468,7 +469,7 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&MEM_Init);
 	secprop->AddInitFunction(&HARDWARE_Init);
 	pint = secprop->Add_int("memsize", when_idle, 16);
-	pint->SetMinMax(1, 384);
+	pint->SetMinMax(MEM_GetMinMegabytes(), MEM_GetMaxMegabytes());
```

Comment:
very nice here, putting the memory module in charge of setting the limits :+1:

Reply:
Nice one, this is something we should adapt going forward 💯